plugin/pkg/admission/imagepolicy/admission.go

```diff
@@ -181,6 +181,20 @@ func (a *Plugin) admitPod(pod *api.Pod, attributes admission.Attributes, review
 		return errors.New("one or more images rejected by webhook backend")
 	}
 
+	if review.Status.Annotations == nil {
+		return nil
+	}
+	reviewAnnotations := a.filterAnnotations(review.Status.Annotations)
```

Comment:
should it be an error if the webhook returns invalid annotations?

Reply:
I can buy either argument (webhooks should be written properly, so we should just reject, or webhooks are trusted, so let's just filter out annotations).

Added error logging, that should be enough to detect a bad configuration.